### PR TITLE
GirlCode page with custom blog links

### PR DESCRIPTION
**Pull request by afkatja**
- [x] The small blocks have to be filled yet (some related organizations, etc)
 - [x] Less stock-photo pictures have to be added with art direction 
 - [x] All the content has to be filled
 - [ ] Copy checks should be done
 - [ ] ~~Image of the girls @ Q42 has to be made (new photos, a group photo?)~~
 - [ ] Maybe more blocks are to be added....

### CODE REVIEW

File: client/girlcode.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Meteor.startup ->
```
Comment (Primigenus) [inline client/girlcode.coffee]: You don't need to specify any `Meteor.startup` stuff to add a page, so you can remove this file.

Comment (afkatja) [inline client/girlcode.coffee]: For local development it's necessary (for me at least)

**Comment by Primigenus** [comment]
You may need to resize some of the images now that the styling for large blocks has changed.

**Comment by Primigenus** [comment]
Here's what I would do with this layout:

- Put the "Girl Code at Q42" block at the top and get rid of the massive Girl Code logo as well as cutting the text down to just one paragraph that fits the size of the associated image.
- Beneath that, have the larger block with sidebar where you can include the other paragraphs that didn't fit with the headline block.
- Then the "first Girl Code meetup" block
- Then finish with the three columns.

I would also add some information about projects women at Q have worked on. I think our Girl Code page should be inspirational as well, and show how we're contributing in terms of work beyond just organising meetups and linking to articles we like.

**Comment by afkatja** [comment]
@Primigenus check je PR nog eens? Jouw laatste comment is beantwoord in de mail.

**Comment by dierat** [comment]
Updated version of this pull request: https://github.com/Q42/q42.nl/pull/134

**Pull request closed without merging**